data/README.md

```diff
@@ -62,3 +62,22 @@ wp.data.query( select => {
 	};
 } )( Component );
 ```
+
+
+### `wp.data.subscribe( listener: func )`
+
+Function used to subscribe to data changes. The listener function is called each time a change is made to any of the registered reducers. This function returns a `unsubscribe` function used to abort the subscription.
+
+```js
+// Subscribe.
+const unsubscribe = wp.data.subscribe( () => {
+	const data = {
+		slug: wp.data.select( "core/editor", "getEditedPostSlug" ),
```

Comment:
Odd we're including examples which violate code style guidelines ;)

>WordPress uses single quotation marks for string declarations.

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/

Reply:
I blame linters and their limitations 😄